src/main/java/org/broadinstitute/hellbender/engine/VariantWalker.java

```diff
@@ -59,6 +53,33 @@ void initializeFeatures() {
         initializeDrivingVariants();
     }
 
+    /**
+     * Overriding the superclass method to preferentially
+     * choose the sequence dictionary from the driving source of variants.
+     *
+     * Walkers may override.
```

Comment:
Untrue -- walkers may not override, since `VariantWalker` marks it as `final` (as it should)